pkg/connectorbuilder/connectorbuilder.go

```
@@ -1037,10 +1045,19 @@
 		b.m.RecordTaskFailure(ctx, tt, b.nowFunc().Sub(start))
 		return nil, fmt.Errorf("error: event feed not implemented")
 	}
+	var eventTypes = request.EventTypes
+	if request.EventTypes == nil || len(request.EventTypes) == 0 {
```

golangci-lint (1.64.8)

S1009: should omit nil check; len() for nil slices is defined as zero

(gosimple)